src/status_im/contexts/wallet/create_account/select_keypair/view.cljs

```diff
@@ -26,25 +28,52 @@
       :accessibility-label :import-private-key
       :label               (i18n/label :t/import-private-key)}]]])
 
-(def accounts
-  [{:account-props {:customization-color :turquoise
-                    :size                32
-                    :emoji               "\uD83C\uDFB2"
-                    :type                :default
-                    :name                "Trip to Vegas"
-                    :address             "0x0ah...71a"}
-    :networks      [{:network-name :ethereum :short-name "eth"}
-                    {:network-name :optimism :short-name "opt"}]
-    :state         :default
-    :action        :none}])
+(defn parse-accounts
+  [given-accounts]
+  (->> given-accounts
+       (filter (fn [{:keys [path]}]
+                 (not (string/starts-with? path constants/path-eip1581))))
+       (map (fn [{:keys [colorId emoji name address]}]
+              {:account-props {:customization-color (if (not-empty colorId) (keyword colorId) :blue)
+                               :size                32
+                               :emoji               emoji
+                               :type                :default
+                               :name                name
+                               :address             address}
+               :networks      [{:network-name :ethereum :short-name "eth"}
+                               {:network-name :optimism :short-name "opt"}
+                               {:network-name :arbitrum :short-name "arb1"}]
+               :state         :default
+               :action        :none}))))
 
-(defn view
+(defn keypair
+  [item index _ {:keys [profile-picture compressed-key]}]
+  (let [main-account (first (:accounts item))
+        color        (keyword (:colorId main-account))
```

Comment:
parsing the data when it's pulled in initially helps avoid keyword handling like this inside the components etc 👍